src/tools/cephfs_mirror/PeerReplayer.cc

```diff
@@ -169,54 +170,59 @@ class PeerReplayerAdminSocketHook : public AdminSocketHook {
+  m_service_daemon->add_or_update_peer_attribute(
+      m_filesystem.fscid, m_peer, SERVICE_DAEMON_RECOVERED_DIR_COUNT_KEY,
+      (uint64_t)0);
+
+  std::string labels = ceph::perf_counters::key_create(
+      "cephfs_mirror_peers",
+      {{"source_fscid", stringify(m_filesystem.fscid)},
+       {"source_filesystem", m_filesystem.fs_name},
+       {"peer_cluster_name", m_peer.remote.cluster_name},
+       {"peer_cluster_filesystem", m_peer.remote.fs_name}});
   PerfCountersBuilder plb(m_cct, labels, l_cephfs_mirror_peer_replayer_first,
-			  l_cephfs_mirror_peer_replayer_last);
+                          l_cephfs_mirror_peer_replayer_last);
   auto prio = m_cct->_conf.get_val<int64_t>("cephfs_mirror_perf_stats_prio");
   plb.add_u64_counter(l_cephfs_mirror_peer_replayer_snaps_synced,
-		      "snaps_synced", "Snapshots Synchronized", "sync", prio);
+                      "snaps_synced", "Snapshots Synchronized", "sync", prio);
   plb.add_u64_counter(l_cephfs_mirror_peer_replayer_snaps_deleted,
-		      "snaps_deleted", "Snapshots Deleted", "del", prio);
+                      "snaps_deleted", "Snapshots Deleted", "del", prio);
   plb.add_u64_counter(l_cephfs_mirror_peer_replayer_snaps_renamed,
-		      "snaps_renamed", "Snapshots Renamed", "ren", prio);
+                      "snaps_renamed", "Snapshots Renamed", "ren", prio);
   plb.add_u64_counter(l_cephfs_mirror_peer_replayer_snap_sync_failures,
-		      "sync_failures", "Snapshot Sync Failures", "fail", prio);
-  plb.add_time_avg(l_cephfs_mirror_peer_replayer_avg_sync_time,
-		   "avg_sync_time", "Average Sync Time", "asyn", prio);
-  plb.add_u64_counter(l_cephfs_mirror_peer_replayer_sync_bytes,
-		      "sync_bytes", "Sync Bytes", "sbye", prio);
+                      "sync_failures", "Snapshot Sync Failures", "fail", prio);
+  plb.add_time_avg(l_cephfs_mirror_peer_replayer_avg_sync_time, "avg_sync_time",
+                   "Average Sync Time", "asyn", prio);
+  plb.add_u64_counter(l_cephfs_mirror_peer_replayer_sync_bytes, "sync_bytes",
+                      "Sync Bytes", "sbye", prio);
   plb.add_time(l_cephfs_mirror_peer_replayer_last_synced_start,
-	       "last_synced_start", "Last Synced Start", "lsst", prio);
-  plb.add_time(l_cephfs_mirror_peer_replayer_last_synced_end,
-	       "last_synced_end", "Last Synced End", "lsen", prio);
+               "last_synced_start", "Last Synced Start", "lsst", prio);
+  plb.add_time(l_cephfs_mirror_peer_replayer_last_synced_end, "last_synced_end",
+               "Last Synced End", "lsen", prio);
   plb.add_time(l_cephfs_mirror_peer_replayer_last_synced_duration,
-	       "last_synced_duration", "Last Synced Duration", "lsdn", prio);
+               "last_synced_duration", "Last Synced Duration", "lsdn", prio);
   plb.add_u64_counter(l_cephfs_mirror_peer_replayer_last_synced_bytes,
-		      "last_synced_bytes", "Last Synced Bytes", "lsbt", prio);
+                      "last_synced_bytes", "Last Synced Bytes", "lsbt", prio);
```

Comment:
Could you please avoid these cleanup in this PR, which makes hard to review? Let's do it in another file cleanup PR if necessary. This is an unnecessary git history change too. So let's avoid it.